Formula/r/rapidyaml.rb

```diff
@@ -0,0 +1,27 @@
+class Rapidyaml < Formula
+  desc "Library to parse and emit YAML, and do it fast"
+  homepage "https://github.com/biojppm/rapidyaml"
+  url "https://github.com/biojppm/rapidyaml/releases/download/v0.7.2/rapidyaml-0.7.2-src.tgz"
+  sha256 "175b71074005a7d48ae03e973f1a7a5c104c588f193ba85044c5b4a97341aae0"
+
+  depends_on "cmake" => :build
+
+  conflicts_with "c4core", because: "both install `c4core`"
+
+  def install
+    system "cmake", ".", "-B", "builddir", "-DCMAKE_INSTALL_PREFIX=#{prefix}", "-DBUILD_SHARED_LIBS=ON"
+    system "cmake", "--build", "builddir"
+    system "cmake", "--install", "builddir"
+  end
```

Comment:
this is more common code style we do in homebrewc-ore

```suggestion
  def install
    system "cmake", "-S", ".", "-B", "build", "-DBUILD_SHARED_LIBS=ON", *std_cmake_args
    system "cmake", "--build", "build"
    system "cmake", "--install", "build"
  end

```